alpha/external-web/Dockerfile

```diff
@@ -8,4 +8,20 @@ COPY . /usr/src/app
 
 EXPOSE 3000
 
+# Install PDF Toolkit. Required for PDF compression.
+ENV PDFTK_VERSION 2.02
+
+RUN apt-get update -y && \
+    apt-get upgrade -y && \
```

Comment:
do we have to update and upgrade each time if we are installing the same version on the next line? (4.8)